File: src/sdk-reference/php/3/memory-storage/zadd/snippets/zadd-1.php

<?php

use \Kuzzle\Kuzzle;


$kuzzle = new Kuzzle('localhost');

$elements = [
  ['score' => 1, 'member' => 'foo'],
  ['score' => 2, 'member' => 'bar'],
  ['score' => 3, 'member' => 'baz']
];

try {
  $value = $kuzzle->memoryStorage()->zadd('key', elements);
}
catch (ErrorException $e) {

}

File: src/sdk-reference/android/3/memory-storage/zadd/snippets/zadd-1.java

JSONObject[] elements = new JSONObject[]{
  new JSONObject().put("score", 1).put("member", "foo"),
  new JSONObject().put("score", 2).put("member", "bar"),
  new JSONObject().put("score", 3).put("member", "baz")
};

kuzzle.memoryStorage.zadd("key", elements new ResponseListener<Long>() {
  public void onSuccess(int count) {
    // callback called once the action has completed
  }

  @Override
  public void onError(JSONObject error) {
  }
});
